### PR TITLE
Hoc2022 - Code.org homepage banner - Nov launch

### PR DESCRIPTION
**Pull request by kelbyhawn**
Update the banner on the https://code.org homepage for the November Hour of Code launch.
- The string "The Hour of Code is coming" will show for `soon-hoc` 
- The string "The Hour of Code is here" will show for `actual-hoc` 

**Relevant info:** [Asana task](https://app.asana.com/0/1202764401306810/1202893069545210/f)

**Related PRs:** https://github.com/code-dot-org/code-dot-org/pull/48292

## Desktop
Desktop

## Tablet
Tablet

## Mobile
Mobile

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/css/hoc-banner.scss [hunk 1]
```diff
@@ -1,48 +1,110 @@
 @import 'color.scss', 'font.scss', 'imports/_breakpoints.scss';
 
 // Code.org HoC homepage banner
-.hoc-banner.homepage {
-  text-align: center;
-  color: $darkest_gray;
-  padding: 7em 0;
-  line-height: 1.4;
+#bars {
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/css/hoc-banner.scss:4]: This code is referencing this: https://github.com/code-dot-org/code-dot-org/blob/c03e5c64dc0631d29380b536f70e705baa05da9d/pegasus/sites.v3/code.org/views/homepage_hero.haml#L95-L99

File: pegasus/sites.v3/code.org/homepage.json [hunk 2]
```diff
@@ -18,14 +20,50 @@
...
+      "hoc_modes": ["actual-hoc"],
+      "class": "hoc-banner homepage",
+      "desktopImage": "/images/hoc2022-codeorg-hero-banner-02.png",
+      "mobileImage": "/images/hoc2022-codeorg-hero-banner-mobile-02.png",
+      "leftImage": "/images/hoc2022-codeorg-hero-banner-02-left.svg",
+      "rightImage": "/images/hoc2022-codeorg-hero-banner-02-right.svg",
+      "actions": [
+        {
+          "type": "text",
+          "text": "homepage_hero_hoc_is_here"
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/homepage.json:49]: I made a duplicate of the above banner so we can swap out "The Hour of Code is coming" with "The Hour of Code is here" during `actual-hoc`.

File: pegasus/sites.v3/code.org/public/css/hoc-banner.scss [hunk 3]
```diff
@@ -1,48 +1,109 @@
...
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/css/hoc-banner.scss:4]: The code in this section refers to this: https://github.com/code-dot-org/code-dot-org/blob/848673c33968f431efe4d9c1d13ebf583db0bed9/pegasus/sites.v3/code.org/views/homepage_hero.haml#L95-L99

File: pegasus/sites.v3/code.org/public/css/hoc-banner.scss [hunk 4]
```diff
@@ -1,48 +1,110 @@
...
-    padding: 9em 0;
+  @media screen and (max-width: $width-md) {
+    height: 400px;
   }
 
-  @media screen and (max-width: $width-lg)  {
-    padding: 7em 0 6em;
-  }
+  & > div.col-33 {
+    position: absolute;
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/css/hoc-banner.scss:13]: I used absolute positioning here to allow the middle section to be wider than 33% and overlap the left and right images so they could maintain their size w/o overwriting anything in [homepage_hero.haml](https://github.com/code-dot-org/code-dot-org/blob/848673c33968f431efe4d9c1d13ebf583db0bed9/pegasus/sites.v3/code.org/views/homepage_hero.haml).

**Review by davidsbailey — APPROVED** [review]
looks great! nice screenshots 👍

**Pull request merged**